HLTriggerOffline/Scouting/python/PatElectronTagProbeAnalyzer_cfi.py

```diff
@@ -1,9 +1,55 @@
 import FWCore.ParameterSet.Config as cms
 from DQMServices.Core.DQMEDAnalyzer import DQMEDAnalyzer
 
+DoubleEGL1 = [
+  "L1_DoubleEG_LooseIso16_LooseIso12_er1p5",
+  "L1_DoubleEG_LooseIso18_LooseIso12_er1p5",
+  "L1_DoubleEG_LooseIso20_LooseIso12_er1p5",
+  "L1_DoubleEG_LooseIso22_LooseIso12_er1p5",
+  "L1_DoubleEG11_er1p2_dR_Max0p6"
```

Comment:
@tihsu99, @silviodonato, @elfontan 

In https://its.cern.ch/jira/browse/CMSHLT-3488 we removed `L1_DoubleEG11_er1p2_dR_Max0p6` from the HLT menu (thus it won't be present in any 2025 data). Can you follow-up by cleaning this seed from the configurations?